Build/SaveLoadScreen.cc

```diff
@@ -348,9 +381,89 @@ static void SelectedSaveRegionCallBack(MOUSE_REGION* pRegion, INT32 iReason);
 static void SelectedSaveRegionMovementCallBack(MOUSE_REGION* pRegion, INT32 reason);
 static void StartFadeOutForSaveLoadScreen(void);
 
+static void CreateLoadscreenTab()
+{
+	MakeTab(0,        20, BtnSlgNormalGameTabCallback, gs_dead_is_dead_mode_tab_name[0]);
+	MakeTab(1, 90, BtnSlgDeadIsDeadTabCallback,    gs_dead_is_dead_mode_tab_name[1]);
+	// Render the Normal Tab as selected after create
+	giLoadscreenTab[SLS_TAB_NORMAL]->uiFlags |= BUTTON_CLICKED_ON;
+}
+
+static void RemoveLoadscreenTab()
+{
+	for (int i = 0; i < SLS_TAB_LENGTH; i++)
+	{
+		RemoveButton(giLoadscreenTab[i]);
+		UnloadButtonImage(giLoadscreenTabButtonImage[i]);
+	}
+}
+// This function determines which tab to activate in the load Screen.
+// Depending on:
+// In which tab is the last save
+// Are there any saves in the tab
+// It also deactivates tabs with no saves
+static void selectActiveTab()
+{
+	INT8 const lastSaveInTab = (INT8) gGameSettings.bLastSavedGameSlot / NUM_SAVE_GAMES;
+	for (INT8 i = 0; i < NUM_SAVE_GAMES_TABS; i++)
+	{
+		gfDiDTab = i; // Not strictly correct, as gfDiDTab is a boolean. This prepares the function for possible additional tabs without breaking the current Tab implementation.
```

Comment:
Why not make it an `UINT8` then? Also I would rename it to `selectedTab` or something like that.